yarn-project/stdlib/src/contract/artifact_hash.ts

```diff
@@ -6,7 +6,8 @@ import { MerkleTree, MerkleTreeCalculator } from '@aztec/foundation/trees';
 
 import { type ContractArtifact, type FunctionArtifact, FunctionSelector, FunctionType } from '../abi/index.js';
 
-const VERSION = 1;
+// TODO: should this be redefined here as constant??
+const ROLLUP_VERSION = 1;
```

Comment:
This was not intended to be the rollup version. The idea was being able to version artifact formats, so this would be more like an `ARTIFACT_DEFINITION_VERSION`.

Reply:
Same happens in `stdlib/src/contract/interfaces/contract_class.ts`

Reply:
An right on